src/redux/actions/organisationUnits.js

```diff
@@ -40,9 +40,12 @@ export const loadOrganisationUnits = () => dispatch =>
             )
         })
 
-export const selectOrganisationUnit = (_event, { id, path, displayName }) => ({
+export const selectOrganisationUnit = (
+    _event,
```

Comment:
I think the `_event` parameter should be removed from this function, the mapDispatchToProps function should handle this (why should a function accept a parameter it doesn't use)

Reply:
Yes, you are right. I spotted before what you were doing in the `mapStateToProps` functions, and that is indeed much more elegant. But that would be a change for when we are working on the component/action/reducer. And I will do so once your work has been merged.